### PR TITLE
GH-13404: Add 3 test cases for removePermissionsCmdF

### PR DESCRIPTION
**Pull request by vesari**
#### Summary

I added 3 unit tests for the removePermissionsCmdF function.

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/13404

### CODE REVIEW

**Comment by vesari** [comment]
Were you able to review my PR yet? Anything I should do?

**Comment by jasonblais** [comment]
Hi @vesari, apologize for the delay. Both Miguel and Mario have been out for the Christmas and New Year's holidays, but should be back this week. I'll remove the stale label.

cc @mgdelacroix / @ethervoid to help with reviews when back :)

File: commands/permissions_test.go [hunk 1]
```diff
@@ -54,3 +54,103 @@ func (s *MmctlUnitTestSuite) TestAddPermissionsCmd() {
...
+			PatchRole(mockRole.Id, expectedPatch).
+			Return(&model.Role{}, &model.Response{Error: nil}).
+			Times(1)
+
+		args := []string{mockRole.Name, "delete"}
+		err := removePermissionsCmdF(s.client, &cobra.Command{}, args)
+		s.Require().Nil(err)
+	})
+
+	s.Run("Removing an existing permission from an existing role", func() {
```
Comment (ethervoid) [inline commands/permissions_test.go]: Maybe I'm missing something here but it looks like you're testing the same twice (this test seems to check for the same than the previous one) If that the case, I think this test should be removed.

I'd modify this test or a need one that checks for multiple permission removal in order to test that use case

File: commands/permissions_test.go [hunk 2]
```diff
@@ -54,3 +54,103 @@ func (s *MmctlUnitTestSuite) TestAddPermissionsCmd() {
...
+		mockError := model.NewAppError("Role", "role_not_found", nil, "", http.StatusNotFound)
+		s.client.
+			EXPECT().
+			GetRoleByName(mockRole.Name).
+			Return(nil, &model.Response{Error: mockError}).
+			Times(1)
+
+		args := []string{mockRole.Name, "delete"}
+		err := removePermissionsCmdF(s.client, &cobra.Command{}, args)
+		s.Require().NotNil(err)
```
Comment (ethervoid) [inline commands/permissions_test.go]: Please add a checkfor the returned Error as well

**Comment by vesari** [comment]
@ethervoid thank you for your feedback, I've tried to incorporate your suggestions. Please have another look :)

**Pull request merged**